app/views/users/loginpage.html.erb

```diff
@@ -0,0 +1,11 @@
+<%= form_for(:user, url: login_path, method: :post) do |f| %>
+  <a><%= f.label :name %></a>
+  <a><%= f.text_field :name %></a>
+
+  <a><%= f.label :password %></a>
+  <a><%= f.text_field :password %></a>
```

Comment:
can we change this to `password_field` to ensure security when entering password? It won't show up what the user is entering in for their password, it will show up as black dots instead

Reply:
^^seconded